tests/std/tests/P0645R10_text_formatting_utf8/test.cpp

```diff
@@ -82,8 +82,8 @@ void test_width_estimation() {
         {TYPED_LITERAL(CharT, "\u2e80\x58"), 3},
         {TYPED_LITERAL(CharT, "\u303e\x58"), 3},
         {TYPED_LITERAL(CharT, "\u303f\x58"), 2},
-        {TYPED_LITERAL(CharT, "\u3040\x58"), 3},

```

Comment:
The old comment(`defined in n4885`) doesn't apply to new value. I think we should instead comment out these test cases with citation to new standard or the paper.

Reply:
The citation should be updated to the latest WP, and the test cases should reflect the latest WP, without having anything commented out. We continuously target the latest WP and we never care about older versions (except insofar as we support C++14/17/20/23 etc. modes - but the changes in this PR are unconditional back to C++20 so there are no modes to worry about).